vector/src/main/res/layout/item_pushgateway.xml

```diff
@@ -130,5 +129,11 @@
         android:textStyle=""
         tools:text="event_id_only" />
 
+    <Button
```

Comment:
Format and URL are not relevant for Email, can you fix that please? (Not really part of the PR, but a good opportunity to make it right):
<img width="65" alt="image" src="https://user-images.githubusercontent.com/3940906/134491778-c472e9b2-2a23-49d6-8c65-f4331f9c0233.png">

Reply:
1865e3c450483be7e43d0a399ba767493bbd4850

| HTTP | EMAIL |
| --- | --- |
|![device-2021-09-23-125643](https://user-images.githubusercontent.com/1848238/134502757-37d48d64-632f-4b9e-a46a-fb38592a41ee.png)|![device-2021-09-23-125652](https://user-images.githubusercontent.com/1848238/134502754-5e31632c-06e4-4831-a3db-3d61ea2053d7.png)

Reply:
thanks!